easybuild/easyconfigs/m/motif/motif-2.3.8-GCCcore-13.2.0.eb

```
@@ -0,0 +1,41 @@
+easyblock = 'ConfigureMake'
+
+name = 'motif'
+version = '2.3.8'
+
+homepage = 'https://motif.ics.com/'
+description = """Motif refers to both a graphical user interface (GUI) specification and the widget toolkit for building
+ applications that follow that specification under the X Window System on Unix and other POSIX-compliant systems.
+ It was the standard toolkit for the Common Desktop Environment and thus for Unix."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['859b723666eeac7df018209d66045c9853b50b4218cecadb794e2359619ebce7']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.8.2'),
+    ('binutils', '2.40'),
+    ('util-linux', '2.39'),
+]
+
+dependencies = [
+    ('X11', '20231019'),
+    ('libpng', '1.6.40'),
+    ('freetype', '2.13.2'),
+    ('libjpeg-turbo', '3.0.1'),
+    ('bzip2', '1.0.8'),
+]
+
+# makefile is not parallel safe
+parallel = 1
```

Use `maxparallel` (since `parallel` easyconfig parameter was deprecated in EasyBuild 5.0):

```suggestion
maxparallel = 1